core/arch/opnd.h

```diff
@@ -842,6 +842,11 @@ enum {
 #    endif
 
     DR_NUM_GPR_REGS = DR_REG_STOP_GPR - DR_REG_START_GPR + 1,
+#    ifdef AARCH64
+    DR_NUM_SIMD_VECTOR_REGS = DR_REG_Z31 - DR_REG_Z0 + 1,
```

Comment:
(Should we have START_SIMD and STOP_SIMD too?  Maybe not worth it.)

Reply:
I'd leave this to the ARM guys, not part of this cl/ for sure.